src/main/webapp/index.html

```diff
@@ -25,19 +26,9 @@ <h1>Language Sentiment Analyzer</h1>
           <h2>How it works</h2>
           <p>In the form below, you can input a piece of text <u>in Spanish</u> and the analyzer will translate it and understand its sentiment score.</p>
           <p>The score ranges from -1.0 to 1.0 with a negative score indicating a negative sentence and a positive score indicating a positive sentence.</p>
-          <p>Now go on and give it a try!</p>
+          <p>Now go on and give it a try! <a href="/translate_sentiment.html">Try it here!</a></p>
```

Comment:
I thought we were keeping the sentiment translator on the home page? Otherwise LGTM!!

Reply:
We can if we want! Otherwise, I'm fine with either other. I think having it on a separate webpage can allow for us to have a better homepage.

Reply:
Awesome lets go with that then.